.github/workflows/ci.yml

```diff
@@ -125,7 +129,7 @@ jobs:
 
     # To return the correct result even though we set 'continue-on-error: true'
     # - name: Coerce correct build result
-    #   if: matrix.os != 'windows-latest' && steps.step_test.outputs.status != ${{ matrix.SUCCESS }}
+    #   if: matrix.os != 'windows' && steps.step_test.outputs.status != ${{ matrix.SUCCESS }}
     #   run: |
     #     echo "step_test ${{ steps.step_test.outputs.status }}\n"
     #     exit 1
```

Comment:
Do we need this? I forget why we have it commented

Reply:
I think it was for test coverage reports? We've never really turned them on since v2 betas I think. I'm not sure if we ever will. But still, harmless in case we eventually do, but also we can find it pretty easily in the commit logs if we need to bring it back. So up to you 🤷‍♂️